app/features/feedback/shared/components/checkbox-input.jsx

```diff
@@ -0,0 +1,34 @@
+import React, { PropTypes } from 'react';
+import uuidv4 from 'uuid/v4';
+
+
+function CheckboxInput({ title, help, onChange, checked, name }) {
+  const uuid = uuidv4();
+  return (
+    <div>
```

Comment:
I think using `<fieldset>` is a bit more semantic than a div here, but the default browser styles for fieldset may not be desirable.

Reply:
ARGH SUCH BORDERS

Reply:
<img width="545" alt="screenshot 2017-12-04 18 57 24" src="https://user-images.githubusercontent.com/2725841/33570552-196c57e0-d925-11e7-892d-b14fd47ef461.png">